xml/ha_config_basics.xml

```diff
@@ -698,6 +694,11 @@
    <sect3 xml:id="sec.ha.config.basics.resources.advanced.masters">
     <title>Multi-state Resources</title>
     <para>
+    <remark>taroth 2018-03-09: ygao, I guess this needs to be updated.
+    Could you suggest a new text, please? Also, the 'Pacemaker
+    Explained' Guide from upstream has been deprecated. Is there a new
+    place where one can find documentation on promotable clones (aka multi-state
+    resources) upstream?</remark>
      Multi-state resources are a specialization of clones. They allow the
      instances to be in one of two operating modes (called
      <literal>master</literal> or <literal>slave</literal>, but can mean
```

Comment:
AFAICS, "master" and "slave" are still used in terms of describing  state of resource instances. I think we can keep this until anything else probably like "primary/secondary" has been explicitly chosen instead to shown from crm_mon.

Reply:
Good news, thanks. :)